### DISCUSSION TITLE
When to document new features

### DISCUSSION THREAD

**Discussion by estelle** [mdn/mdn-community#488] [Content] [answered]
On  https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/What_we_write, the following is written:

>If a feature is implemented by multiple browsers and either accepted as standard or is progressing towards standardization, then yes, we definitely document it here.

Of course! it goes on:

> If a feature is still very experimental and not implemented in multiple browsers and/or liable to change, then it is still suitable for inclusion, but may not be seen as a priority for the writing team to work on.

Browsers that implement new features generally want to document them, and will document them. My question surrounds deciding when to include that documentation. The page continues:

> In other words, web technologies to be documented on MDN Web Docs should fulfil the following criteria:

> Be on a standards track.
> Be specified in a specification published by a reliable standards body.
> Be implemented by at least one rendering engine.
> Be released in a stable browser version.

the issue is, there is no "and" or "or" in that list. I think it should be "and" but we've been doing "or". 

see https://github.com/mdn/content/pull/29830

**Comment by hamishwillee** [comment, marked as answer]
There is a "when" section further down: https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/What_we_write#when_we_document_a_new_technology

>... try to strike a balance between publishing the documentation early enough so that developers can learn about new features as soon as they need to and publishing it late enough so that the technology is mature and stable so that the documentation won't need constant updates or rapid removal.

> When the feature is on a standards track and is implemented somewhere.

^^^ That is the priority.

In practice it is more like "Be implemented by at least one rendering engine." AND one of the other lines. Things don't have to be on the standards track or be in stable versions, so those definitely are "OR" (more about prioritization)

**Reply by bsmth** [reply]
I think this covers it. Is there anything to be done on the writing guidelines or are we happy with it as it stands?
